keygen.go

```diff
@@ -67,4 +67,9 @@ var (
 	// requests. Set this to a custom HTTP client, to implement e.g.
 	// automatic retries, rate limiting checks, or for tests.
 	HTTPClient = cleanhttp.DefaultPooledClient()
+
+	// Scheme is the acceptable cryptographic scheme to be used for
+	// verifying signatures for license keys, license/machine files,
+	// and responses. Supported schemes: ed25519, ecdsa-secp256r1.
+	Scheme = "ed25519"
```

Comment:
We really only need this to effect response signatures, because everything else can be inferred or explicitly set, e.g. license files and keys, respectively. Maybe call this `AcceptSignature`?